array_reverse/array-shift.js

```diff
@@ -0,0 +1,14 @@
+const arr = [1, 2, 3, 4, 5];
+const n = 'hi';
+
```

Comment:
I'm guessing you used these to test you code, but these should be deleted from your final submission code